redundant-bmc/src/manager.cpp

```diff
@@ -68,17 +69,24 @@ sdbusplus::async::task<> Manager::startup()
 
     if (!passiveRoleInfo)
     {
-        if (sibling->isBMCPresent())
+        if (sibling.isBMCPresent())
         {
-            co_await sibling->waitForSiblingUp(siblingTimeout);
+            co_await sibling.waitForSiblingUp(siblingTimeout);
+
+            // Sibling service may have died.  Check again.
```

Comment:
This doesn't seem related to the commit?

Reply:
looks like bad rebase, will fix.

Reply:
pushed fix to just this.